src/models/helpers.ts

```diff
@@ -0,0 +1,80 @@
+import { IFaceDetails } from './interface';
```

Comment:
quick question, do we need this file at all? I feel like removeuseless props will probably live in rekkognition model and calculate the scores should live inside the sentiment model no?

Reply:
up to you :) technically speaking the role of the sentiment model is exactly what you just wrote tho - the model that takes care of the data :)